SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -0,0 +1,45 @@
+Summary:        Prebuilt version of ca-certificates package.
+Name:           prebuilt-ca-certificates
+Version:        20200720
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://hg.mozilla.org
+BuildArch:      noarch
+
+%description
+Prebuilt versions of the ca-certificates package with no runtime dependencies.
+
+%package base
+Summary:        Prebuilt version of ca-certificates-base package.
+BuildRequires:  ca-certificates-base
+Conflicts:      ca-certificates
+Conflicts:      ca-certificates-base
+Conflicts:      ca-certificates-microsoft
+
+%description base
+%{summary}
+
+%prep -q
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_datadir}/pki/ca-trust-legacy/
+mkdir -p %{buildroot}%{_sysconfdir}/pki/
+
+install -p -m 644 %{_datadir}/pki/ca-trust-legacy/* %{buildroot}%{_datadir}/pki/ca-trust-legacy/
+install -p -m 644 %{_datadir}/pki/ca-trust-source/* %{buildroot}%{_datadir}/pki/ca-trust-legacy/
```

Comment:
> %{_datadir}/pki/ca-trust-source [](start = 17, length = 32)

The whole `ca-trust-source` directory is not needed. It is only used to generate the files we actually need in the "prebuilts".